commit 0ddd3336bbfca34765bda35e216bb547d396c812 (refs/changes/86/2986/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-11-22T23:13:17+00:00 (1 year, 11 months ago)
    
    TRITON-18: vuln in semver module
    Reviewed by: Orlando Vazquez <orlando@joyent.com>

diff --git a/package.json b/package.json
index 4425d93..91d0fca 100644
--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "restify": "4.3.1",
     "rimraf": "2.2.6",
     "sdc-clients": "10.0.4",
-    "semver": "3.0.1",
+    "semver": "5.4.1",
     "sshpk": "1.13.0",
     "trace-event": "1.2.0",
     "ufds": "1.2.0",
